examples/agb/collection.json

```diff
@@ -116,7 +139,9 @@
       "description": "The MAAP platform is designed to combine data, algorithms, and computational abilities for the processing and sharing of data related to NASA\u2019s GEDI, ESA\u2019s BIOMASS, and NASA/ISRO\u2019s NISAR missions",
       "roles": [
         "processor",
-        "producer"
+        "producer",
+        "host",
+        "licensor"
```

Comment:
MAAP isn't the licensor - the science team is.

Reply:
ok, can you tell me how to specify that in the context of a STAC provider?

Reply:
we decided to not specify the licensor for now since it is not clear who the licensor is - need to follow up and figure out how to do it at the MAAP organization level.